src/test/scala/com/cognite/spark/datasource/EventsRelationTest.scala

```diff
@@ -590,6 +639,35 @@ class EventsRelationTest extends FlatSpec with Matchers with SparkTest {
     assert(idsAfterDelete.length == 0)
   }
 
+  it should "encode and decode RetrieveEvents" in {
```

Comment:
We (currently and possibly never) don't have such tests, since they will/should be caught by the integration tests.

Reply:
removed